test/e2e/toolchain_test.go

```diff
@@ -6,8 +6,8 @@ import (
 
 	"github.com/codeready-toolchain/toolchain-operator/pkg/apis"
 	"github.com/codeready-toolchain/toolchain-operator/pkg/apis/toolchain/v1alpha1"
-	"github.com/codeready-toolchain/toolchain-operator/pkg/che"
-	"github.com/codeready-toolchain/toolchain-operator/pkg/tekton"
+	"github.com/codeready-toolchain/toolchain-operator/pkg/resources/che"
+	"github.com/codeready-toolchain/toolchain-operator/pkg/resources/tekton"
```

Comment:
Suggestion: I don't think we need this re-factoring to introduce parent pkg `/pkg/resources` for existing packages `/pkg/che` and `/pkg/tekton`.  `che` and `tekton` being top level package gives direct access to add anything for this operator under this package.

If we go with above re-factoring of introducing `resource` package then the future package would be like below:
/pkg/resources/tekton
/pkg/resources/che
/pkg/xyz/tekton
/pkg/xyz/che

But rather above I would like to have it as below:
/pkg/che/resource
/pkg/che/xyz
/pkg/tekton/resource
/pkg/tekton/xyz

As we don't have much now in `che` and `tekton` we can just add without having sub-package for now.

Reply:
I'm a bit confused @nurali-techie. Here you suggest that we have the following structure:
```
/pkg/che/resource
/pkg/che/xyz
/pkg/tekton/resource
/pkg/tekton/xyz
```
but earlier in https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350560867 you suggested that all Tekton and Che resources should go in `pkg/assets/` or `/pkg/resources/`.

Reply:
That is only for generate assets.  All generated assets should go to one and only one package .. while all code that we implement should go to separate pkg `che`, `tekton` etc.

Reply:
hmm, ok, let me see if I can provide you with something cleaner, then. But I don't know if we can generate multiple "asset" files with go-bindata into the same target directory without having an error because each generated go file would provide its own `Asset(...)` func.

Reply:
please see my comment in https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350600268